### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/feature/catalog.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#771] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/feature/catalog.html

VuePress Theme Hope 通过 @vuepress/plugin-catalog 提供目录组件和目录页自动生成。 自动目录页生成 该功能是默认启用的，如果文件夹下没有 README.md，主题会为自动为它生成一个目录页。如需禁用，请将 plugins.catalog 设置为 false。 你可以在主题选项中通过 plugins.catal...

https://theme-hope.vuejs.press/zh/guide/feature/catalog.html

**Comment by shengchaohua** [comment]
请问是否可以做到存在README文件的情况下，允许【目录页自动生成】？
背景：目录是英文，需要使用README文件，使用frontmatter.title使导航栏为中文。

**Reply by shengchaohua** [reply]
已解决：在 Markdown 中使用 `<AutoCatalog />` 即可

**Reply by zhuonixian** [reply]
是作者回复的吗 我搞了好久都不出来  裂开  然后看了下有个评论  ..

**Reply by PhyQxx** [reply]
使用<AutoCatalog />没有效果，是需要单独引什么依赖吗

**Reply by Mister-Hope** [reply]
现在的目录组件叫 `<Catalog />`

**Comment by liangjingwen** [comment]
请问能否根据左侧侧边栏的顺序生成目录的顺序？
